site/gdocs/components/HomepageIntro.scss

```diff
@@ -183,7 +183,10 @@
         display: block;
 
         @include sm-only {
-            height: 172px;
+            // All our new featured images are published at 1200px x 630px
+            // Older featured images aren't published at this size, so we need to
+            // crop them to the correct aspect ratio so that they're all flush
+            height: calc(298px / (1200 / 630));
```

Comment:
I think it's nicer to use `aspect-ratio: 1200 / 630` here.
It's supported in _most_ browsers (not all!), but is way nicer to read.